.github/workflows/nightly-release.yml

```diff
@@ -83,7 +83,7 @@ jobs:
     uses: ./.github/workflows/fpga.yml
     with:
       artifact-suffix: -fpga-realtime-etrng-nolog
-      extra-features: slow_tests
+      extra-features: slow_tests,hw-latest
```

Comment:
Can we test both with hw-1.0 and hw-latest? I don't think they've released 1.0 yet...